src/Simplify.elm

```diff
@@ -999,6 +999,9 @@ Destructuring using case expressions
     List.isEmpty (Array.toList array)
     --> Array.isEmpty array
 
+    a = List.length l == 0
+    --> a = List.isEmpty l
```

Comment:
For consistency:
```suggestion
    List.length list == 0
    --> List.isEmpty list
```

Although this fits better in the `## Lists` section, so it should probably be moved